client/components/dashboard/dashboard-service.js

```diff
@@ -1041,7 +1044,7 @@ DashboardService.prototype.addWriter = function() {
  * @export
  */
 DashboardService.prototype.addParam = function() {
-  this.current.model.params.push(new DashboardParam());
+  this.current.model.params.push(new DashboardParam('', ''));
```

Comment:
In general I'd agree, though I don't consider defaulting strings to empty strings to be a problematic case.  It's fine to leave as-is, though; I don't feel strongly about it.